spec/system/sign_in_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe "Sign in" do
+  context "rendering the sign-in help page" do
+    before do
+      payload = {
+        base_path: "/help/sign-in",
```

Comment:
Non-blocking nitpick. This should be `/sign-in` to match the base_path used in the rest of these tests.

Reply:
Oh yeah, d'oh. Thanks!